src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -0,0 +1,318 @@
+2. Synchronise inventory entries coming from an already-existing commercetools project in the form of
+[JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) 
+[InventoryEntry](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntry.java)
+objects to another commercetools project.
+
+3. Build any of the following commercetools [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) update action
+objects given an old inventory entry, represented by a [InventoryEntry](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntry.java),
+and a new inventory entry, represented by a [InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java):
+    - ChangeQuantity
+    - SetRestockableInDays
+    - SetExpectedDelivery
+    - SetSupplyChannel
+    - inventory custom fields update actions:
+        - SetCustomType
+        - SetCustomField
+
+## How to use it?
+
+### How to use `InventorySyncOptions` and `InventorySyncOptionsBuilder`
+
+In order to use the inventory sync an instance of
+[InventorySyncOptions](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/inventories/InventorySyncOptions.java)
+have to be injected. It can be created by using dedicated builder instance of
+[InventorySyncOptionsBuilder](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/inventories/InventorySyncOptionsBuilder.java)
+ 
+In order to instantiate a `InventorySyncOptions`, a `ctpClient` is required:
+
+#### `ctpClient` [Required]
+Defines the configuration of the commercetools project that inventory entries are going to be synced to.
+````java
+// instantiating a ctpClient
+final SphereClientConfig clientConfig = SphereClientConfig.of("project-key", "client-id", "client-secret");
+final CtpClient ctpClient = new CtpClient(clientConfig);
+  
+// instantiating a InventorySyncOptions
+final InventorySyncOptions inventorySyncOptions = InventorySyncOptionsBuilder.of(ctpClient).build();
+````
+
+Additional configuration for the sync can be configured on the `InventorySyncOptions` instance, according to the need
+of the user of the sync:
+
+#### `ensureChannels` [Optional]
+
+Defines an optional field which represents a strategy for handling with inventory entries of missing supply channels.
+By missing supply channels you could consider supply channels of keys that are referenced in provided inventory
+entries list but do not exists in a target CTP project. Having an inventory entry with missing supply channel
+referenced it could be processed in either ways:
+ - If `ensureChannels` is set to `false` such inventory entry will fail to sync.
```

Comment:
Mean no sync and error callback triggered, that's right 54d38cf96a333ef2bd134b1642f94e394a096a58